src/privateStore.js

```diff
@@ -46,12 +46,12 @@ class PrivateStore {
     if (value != null) {
       value = this._encryptEntry(value)
```

Comment:
I don't get it, why should we check if the value is null? Can't we just always do the encryption regardless?

Reply:
So I'm not sure what encrypting null would mean? How is `null` represented as a `Uint8Array`? And when you decrypt, would you get back `null` or something else like the empty string?

Reply:
Ok, that makes sense. I guess then we would need to think about if value is `undefined` also

Reply:
@oed Actually I verified that `(x != null)` returns `true` also if `x` is `undefined.` :smiley:

Reply:
@christianlundkvist `if (value)` works for both `null` and `undefined`. Javascript is weird sometimes.

Reply:
@oed Hmm that would mean that the value `false` would also trigger this. I’m actually wondering if we are even able to encrypt anything that’s not a string? In EIP 1024 I think they’re solving it by only encrypting/decrypting stringified JSON.

Reply:
Right, didn't think of that. So you are saying that we just do
```js
value = JSON.stringify(value)
```
before encrypting, and
```js
value = JSON.parse(value)
```
after decrypting?

That makes sense. We can put this in the `_encryptEntry` and `_decryptEntry` functions.

Reply:
@oed Yes, I think that's a good way of solving it. Not sure how things like `undefined` are stringified, but this way we should be able to handle all the relevant cases.